domains/quizlink.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "gaurav-kamti",
+    "email": "gauravkamtileo@gmail.com"
+  },
+  "record": {
```

Seem like there is a typo
https://docs.is-a.dev/domain-structure/#-records-required
```suggestion
  "records": {
```